themes/navy/source/css/_partial/page.styl

```diff
@@ -65,7 +62,6 @@ note-warn = hsl(0, 100%, 50%)
   text-decoration: none
   color: color-default
   display: block
-  margin-bottom: 40px
```

Comment:
Remove `margin-bottom` for "Back to Top" button, as the `margin-bottom` is already added for toc-inner.